manifest.json

```diff
@@ -639,6 +639,27 @@
               }
             ]
           },
+          "simplificationIncentivizer": {
+            "default": null,
+            "anyOf": [
+              {
+                "default": {},
+                "type": "object",
+                "properties": {
+                  "simplificationRate": {
+                    "default": 100,
```

Comment:
I wonder if we should zero out all defaults to not be opinionated. I think this is the correct thing to do.